cordformation/src/noderunner/kotlin/net/corda/plugins/NodeRunner.kt

```diff
@@ -20,15 +20,33 @@ private val os by lazy {
     else OS.LINUX
 }
 
+private val debugBasePort by lazy {
+    val debugport = System.getProperty("debug.base.port", "generic").toLowerCase(Locale.ENGLISH)
```

Comment:
There are a number of issues here:
- The property name `debug.base.port`does not identify this property as belonging to `runnodes`. Something like `net.corda.noderunner.debug.port` would be better.
- This is an integer property, and so a default value of "generic" makes no sense. It would be better to use:
```kotlin
private val debugBasePort by lazy {
    Integer.getInteger("net.corda.noderunner.debug.port", 5005)
}
```
which handles any possible `NumberFormatException` automatically.